.infra/index.ts

```diff
@@ -408,6 +422,10 @@ if (isAdhocEnv) {
         },
         targetAverageValue: 100_000,
       },
+      spot: {
+        enabled: true,
+        weight: 70,
+      },
```

Comment:
Since personalized digest is quite bursty might be an issue if spot node gets killed mid processing it could cause spikes, so maybe better to have it a bit lower or turn of until we see how it works? At least it looks like that from my understanding. WDYT?

Reply:
Can set it to 50/50. If a spot node is evicted, they'll just get rescheduled onto other nodes, and if no spot nodes are available, it'll use regular nodes.
It's just a best-effort selection

Reply:
I am worried about time it takes from eviction -> node is back up again, since digest processing is time constrained.

Reply:
As I mentioned, it's a best-effort. If no spot-nodes are schedulable, it'll just fallback to regular nodes

Reply:
Also, I am going to add a antiPodAffinity later on, so that it tries to spread pods more evenly, so that not all of the pods ends up on one node.

Reply:
I understand that, worried about the time it takes from node being evicted and then replaced with whatever node is available, is it seconds, minutes...?

Reply:
Seconds